### ISSUE TITLE
Missing requirements

### ISSUE THREAD

**Issue by Guy-Dipietro-Bose** [markolofsen/py_translator#2]
When doing a translation you'll get an error because of uninstalled package:

  File "/Users/guydipietro/PycharmProjects/CastleTestUtils/ctvenv/lib/python2.7/site-packages/CastleTestUtils/VoiceUtils/text_to_speech/text_to_speech.py", line 2, in <module>
    from py_translator import Translator
  File "/Users/guydipietro/PycharmProjects/CastleTestUtils/ctvenv/lib/python2.7/site-packages/py_translator/__init__.py", line 8, in <module>
    from .html_connector import TEXTLIB
  File "/Users/guydipietro/PycharmProjects/CastleTestUtils/ctvenv/lib/python2.7/site-packages/py_translator/html_connector.py", line 1, in <module>
    from .html_translator import superTranslator, massTranslator, setProxy
  File "/Users/guydipietro/PycharmProjects/CastleTestUtils/ctvenv/lib/python2.7/site-packages/py_translator/html_translator.py", line 5, in <module>
    from langdetect import detect
ImportError: No module named langdetect

**Comment by feifanhanmc** [comment]
pip install langdetect

**Comment by Guy-Dipietro-Bose** [comment]
yes, but dependencies should be defined (setup.py, requirements.txt, pyenv).
